views/pulsar/v2/helpers/tabs.html.twig

```diff
@@ -31,9 +31,11 @@
                 {% if tab.disabled is defined and tab.disabled == true %}
                     {% set href = '#' %}
                     {% set type = 'link' %}
+                    {% set tabindex = '-1' %}
                 {% else %}
                     {% set href = '#' ~ tab.id %}
                     {% set type = 'tab' %}
+                    {% set tabindex = '0' %}
```

Comment:
Why do we need to add a tab index of 0 here? Could we not specify a tab index at all if the link is not disabled, as it then should follow sequential flow

Reply:
Another couple of options (which you might have explored) would be:

1. rather than displaying an `a`, display a `span` with similar styles (removing the need for tabindex, css and JS)
2. remove the href completely, therefore the link cannot be focused by keyboard or mouse (no need for JS or tab index)

Reply:
> Why do we need to add a tab index of 0 here? Could we not specify a tab index at all if the link is not disabled, as it then should follow sequential flow

Not specifying the tabindex is the same as tabindex 0, so it felt more obvious for other developers to simply toggle the option between 1/0 as they need to.

> rather than displaying an a, display a span with similar styles (removing the need for tabindex, css and JS)

Using disabled links is fine imho and keeps things simpler for any developer needing to toggle disabled state easily, without having to rewrite markup on the fly.

> remove the href completely, therefore the link cannot be focused by keyboard or mouse (no need for JS or tab index)

Again, I'm thinking of other developers here, they'd have to save href state somehow if loading disabled links rather than simply toggling their disabled state.